sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOptions.cs

```diff
@@ -6,13 +6,17 @@
 namespace Azure.AI.FormRecognizer
 {
     /// <summary>
-    /// The set of options that can be specified when calling a recognize content method
-    /// to configure the behavior of the request.
+    /// The set of options that can be specified when calling a Recognize Content method
+    /// to configure the behavior of the request. For example, specify the content type of the
+    /// form, the language of the form, and the custom page numbers in a multi-page document to analyze.
```

Comment:
`custom page numbers` feels a bit weird to me. I think I'm reading it more like `custom (page numbers)` than `(custom page) numbers`. Maybe something like:
```
...the language of the form, and the page numbers in a multi-page document to analyze.
...the language of the form, and the set of pages in a multi-page document to analyze.
...the language of the form, and the input page numbers in a multi-page document to analyze.
...the language of the form, and which pages in a multi-page document to analyze.
```